software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakNodeDriver.java

```diff
@@ -45,4 +45,6 @@
     void bucketTypeUpdate(String bucketTypeName, String bucketTypeProperties);
 
     void bucketTypeActivate(String bucketTypeName);
+    
+    Integer getNumberOfOpenFiles();
```

Comment:
No need to add it to the interface, usually used when accessed from freemarker templates.